rfai/service/rfai_service.py

```diff
@@ -0,0 +1,13 @@
+from rfai.dao.request_data_access_object import RequestDAO
+from rfai.rfai_status import RFAIStatus
+
+class RFAIService:
+    def __init__(self, repo):
+        self.request_dao = RequestDAO(repo=repo)
+
+    def get_requests(self, status, requester):
+        if status.upper() in RFAIStatus.__members__:
+            request_status = RFAIStatus[status.upper()].value
```

Comment:
can we check if this is best way to iterate an enum?

Reply:
I didn't find other way effective than this.